R/anytime.R

```diff
@@ -175,7 +175,7 @@ anytime <- function(x, tz=getTZ(), asUTC=FALSE, useR=FALSE,
         x <- as.character(x)
     }
 
-    anytime_cpp(x, tz=tz, asUTC=asUTC, useR=useR, oldHeuristic=oldHeuristic)
+    anytime_cpp(x, tz=tz, asUTC=asUTC, asDate=FALSE, useR=useR, oldHeuristic=oldHeuristic)
```

Comment:
That's actually not needed, but does not hurt.

Reply:
And once we have it. we need the same in the two other calls to `anytime_cpp`.  Or in none.

Reply:
But they had the `asDate=` argument already? 

Line 190:
```r
d <- anytime_cpp(x=x, tz=tz, asUTC=asUTC, asDate=TRUE, useR=useR, oldHeuristic=TRUE)
```

Line 214:
```r
 d <- anytime_cpp(x=x, tz=tz, asUTC=TRUE, asDate=TRUE, useR=useR, oldHeuristic=TRUE)
```

Reply:
Oh in which case we're done indeed.  My bad -- we're good.